overlord/snapstate/booted.go

```diff
@@ -94,7 +89,9 @@ func UpdateRevisions(ovld *overlord.Overlord) error {
 		for snapName, snapState := range installed {
 			if name == snapName {
 				if rev != snapState.Current {
-					ts, err := snapstate.RevertToRevision(st, name, rev, snapstate.Flags(0))
+					// FIXME: check that there is no task
```

Comment:
Let's please add a card to the RC to fix that. This sounds like the sort of thing that may break down a device permanently in the wild.